src/cheri/attributes.adoc

```diff
@@ -95,61 +95,107 @@ endif::[]
 //ISA naming
 :CADD:         ADDY
 :CADD_LC:      addy
+:CADD_DESC:    Capability pointer increment
+
 :CADDI:        ADDIY
 :CADDI_LC:     addiy
-:SCADDR:       YWADDR
-:SCADDR_LC:    ywaddr
-:SCBNDS:       YWBNDS
-:SCBNDS_LC:    ywbnds
-:SCBNDSI:      YWBNDSI
-:SCBNDSI_LC:   ywbndsi
-:SCBNDSR:      YWBNDSR
-:SCBNDSR_LC:   ywbndsr
-:GCTAG:        YRTAG
-:GCTAG_LC:     yrtag
-:GCPERM:       YRPERM
-:GCPERM_LC:    yrperm
-:GCHI:         YRHI
-:GCHI_LC:      yrhi
-:GCBASE:       YRBASE
-:GCBASE_LC:    yrbase
-:GCTYPE:       YRTYPE
-:GCTYPE_LC:    yrtype
-:GCMODE:       YRMODE
-:GCMODE_LC:    yrmode
-:GCLEN:        YRLEN
-:GCLEN_LC:     yrlen
-:SCHI:         YWHI
-:SCHI_LC:      ywhi
+:CADDI_DESC:   Capability pointer increment by immediate
+
+:SCADDR:       YADDRW
+:SCADDR_LC:    yaddrw
+:SCADDR_DESC:  Write capability address
+
+:SCBNDS:       YBNDSW
+:SCBNDS_LC:    ybndsw
+:SCBNDS_DESC:  Write capability bounds
+
+:SCBNDSI:      YBNDSIW
+:SCBNDSI_LC:   ybndsiw
+:SCBNDSI_DESC: Write capability bounds by immediate
+
+:SCBNDSR:      YBNDSRW
+:SCBNDSR_LC:   ybndsrw
+:SCBNDSR_DESC: Write capability bounds, rounding up if required
+
+:GCTAG:        YTAGR
+:GCTAG_LC:     ytagr
+:GCTAG_DESC:   Read {ctag}
+
+:GCPERM:       YPERMR
+:GCPERM_LC:    ypermr
+:GCPERM_DESC:  Read capability permissions
+
+:GCHI:         YHIR
+:GCHI_LC:      yhir
+:GCHI_DESC:    Read capability metadata
+
+:GCBASE:       YBASER
+:GCBASE_LC:    ybaser
+:GCBASE_DESC:  Read capability base address
+
+:GCTYPE:       YTYPER
+:GCTYPE_LC:    ytyper
+:GCTYPE_DESC:  Read capability type
+
+:GCMODE:       YMODER
+:GCMODE_LC:    ymoder
+:GCMODE_DESC:  Read capability mode
+
+:GCLEN:        YLENR
+:GCLEN_LC:     ylenr
+:GCLEN_DESC:   Read capability length
+
+:SCHI:         YHIR
+:SCHI_LC:      yhiw
+:SCHI_DESC:    Write capability metadata and clear {ctag}
+
 :SCEQ:         SYEQ
 :SCEQ_LC:      syeq
+:SCEQ_DESC:    Capability equality comparison including {ctag}
+
 :SCSS:         YLT
 :SCSS_LC:      ylt
-:SCMODE:       YWMODE
-:SCMODE_LC:    ywmode
+:SCSS_DESC:    Capability less than comparison including {ctag}
+
+:SCMODE:       YMODEW
+:SCMODE_LC:    ymodew
+:SCMODE_DESC:  Set capability execution mode
+
+:C_MV_CAP:     C.MV
+:C_MV_CAP_LC:  c.mv
+:C_MV_CAP_DESC: Capability register copy, 16-bit encoding
```

Comment:
I don't think we can use this one since it  is confusing  compared to `mv` for integers. The addi*sp ones are non ambiguous since we are adding to so so it needs to be a pointer operand but this one I do think we need to rename.

Reply:
or maybe you can argue that C.ADDI4SPN, C.ADDI16SP will be listed as ADDYI